sharding/README.md

```diff
@@ -290,17 +289,17 @@ function addHeader(int _shardId, uint _expectedPeriodNumber, bytes32 _periodStar
 }
 ```
 
-The `addHeader` function is the most importance in the VMC as it is what provides on-chain verification of collation headers immediately and maintains a canonical ordering of processed collation headers by firing a CollationAdded event log.
+The `addHeader` function is the most important function in the SMC as it provides on-chain verification of collation headers, and maintains a canonical ordering of processed collation headers.
 
 Our current [solidity implementation](https://github.com/prysmaticlabs/geth-sharding/blob/master/sharding/contracts/validator_manager.sol) includes all of these functions along with other utilities important for the our Ruby Release sharding scheme.
```

Comment:
change link to `https://github.com/prysmaticlabs/geth-sharding/blob/master/sharding/contracts/sharding_manager.sol`